components/Molecules/Branding/Header.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="header">
+    <img v-if="logo" :src="logo" class="flex"/>
+    <span v-else class="text-6xl text-white uppercase">{{ name }}</span>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { useBrandingStore } from "~~/stores/branding";
```

Comment:
@aabelmann Zoals aangegeven in een vorige comment, graag overal dezelfde 'prefix' gebruken bij imports